src/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef } from 'react';
+import { DateObject, DatePickerProps } from 'react-multi-date-picker';
+import TextInput from '../TextInput';
+import localeCalendar from './locale_calendar.json';
+import { CustomCalendar } from './styles';
+
+interface Props extends DatePickerProps {
+  value: DateObject;
+  onChange: (date: DateObject) => void;
+  minDate?: Date;
+  maxDate?: Date;
+  disableMonthPicker?: boolean;
+  disableYearPicker?: boolean;
+  workDays?: boolean;
+  holidays?: string[];
+}
+
+const DatePickerComponent: React.FC<Props> = ({
+  value,
+  onChange,
+  minDate,
+  maxDate,
+  disableMonthPicker,
+  disableYearPicker,
+  workDays,
+  holidays,
+}) => {
+  const datePickerRef = useRef();
+  return (
+    <CustomCalendar
+      locale={localeCalendar}
+      className="custom-date-picker"
+      ref={datePickerRef}
+      render={(value, openCalendar, onChange) => {
+        return (
+          <TextInput
+            className="input-date"
+            name="date"
+            id="date"
```

Comment:
Penso que deviriamos receber esse id via props, pode conflitar com um id igual na mesma tela.

Imagine o cenário com 2 inputs desse na mesma tela, um para data início e outro para data fim